src/Components/EmailNotificationForm/EmailNotificationForm.js

```diff
@@ -1,18 +1,24 @@
-import { Component } from "react";
-import "./EmailNotificationForm.css";
-import { states } from "./states"
+import { Component, Fragment } from 'react';
+import './EmailNotificationForm.css';
+import Loading from '../Loading/Loading';
+import Error from '../Error/Error';
+import { states } from './states';
+import { postNewEmailSubscriber } from '../../apiCalls.js';
 
 class EmailNotificationForm extends Component {
   constructor() {
     super();
     this.state = {
-      first_name: "",
-      last_name: "",
-      state_name: "Select",
-      email: "",
-      language: "",
+      first_name: '',
+      last_name: '',
+      state_name: 'Select',
+      email: '',
+      language: '',
       agree_to_emails: false,
-      displayMissingInput: false
+      displayMissingInput: false,
+      isSubmitting: false,
+      successMessage: '',
+      error: ''
```

Comment:
Awesome additions to the form's state. We'll definitely need these properties to conditionally render the success/failure of the api call.

Reply:
That works for me.  I've heard that issues can arise from setting a property to null when it will eventually hold another variable type (string in this case), but I have used null in the past and not experienced any issues.